### ISSUE TITLE
Post comments widget's content and styles

### ISSUE THREAD

**Issue by TheVoidCodes** [elementor/elementor#10628]
The Post Comments widget doesn't have the access to change the style of fields and the button, and it doesn't have the access to change, for example, the content of the button.

![image](https://user-images.githubusercontent.com/27782015/75098536-a883d200-5574-11ea-8e97-a824241836bb.png)

We, users of your plugin, wanna have control over all widgets' content and styles.

**Comment by oooh-boi** [comment]
I just posted [something similar](https://github.com/elementor/elementor/issues/10846). The truth is that Post Comments DOES have its own interface but for some really f**ked reason is not available to users.
Ain't that stupid?

**Comment by isaactchurch** [comment]
Same issue here. I really wish this would be resolved soon.

**Comment by kneops** [comment]
Same request. Everything is editable, but  not the comments. I only have one 'style' and that is not editable!

**Comment by acornavi** [comment]
This is totally missing. Also, to be able to properly style it and make comment elements visible or hidden with default text and messages to be editable.

**Comment by VenkmanNL** [comment]
Not a lot of users need a comment section but some do! And those need to be costumisable into something pretty.

+1

**Comment by oooh-boi** [comment]
I have created an extension for styling the Comments. The extension is a part of "Steroids for Elementor" add-on (plugin) which can be downloaded from the official WP plugins repo (https://wordpress.org/plugins/ooohboi-steroids-for-elementor/).
Cheers!

**Comment by ParkGlobal** [comment]
> I have created an extension for styling the Comments. The extension is a part of "Steroids for Elementor" add-on (plugin) which can be downloaded from the official WP plugins repo (https://wordpress.org/plugins/ooohboi-steroids-for-elementor/).
> Cheers!

Hey, thanks for this! It's great! However, seems it doesn't allow for editing things like the "Post Comment" button colors and such, huh? I was able to adjust that with overall Site Settings in Elementor but would be awesome to be able to control that independently.

**Comment by oooh-boi** [comment]
@ParkGlobal I didn't do that on purpose because - as you wrote - button styles should be inherited from the Site Settings. But I'm "patching" the main add-on occasionally and I might include the button styles panel too.

**Comment by ibqn** [comment]
@oooh-boi, thanks, why is your plugin not on available on github?

**Comment by oooh-boi** [comment]
I literally don't have time to maintain another "project"...

**Comment by Ayadorim** [comment]
Same problem here. I'm not a CSS expert but I made a work around to at least change the colors of the Submit-button. Go to the Elementor Site Settings -> Custom CSS. Copy this CSS in there and change the colors to taste:

.submit {
    color: black;
    border-color: black;
}

.submit:hover {
    background-color: #1B99BF
}

**Comment by isaactchurch** [comment]
> Same problem here. I'm not a CSS expert but I made a work around to at least change the colors of the Submit-button. Go to the Elementor Site Settings -> Custom CSS. Copy this CSS in there and change the colors to taste:
> 
> .submit {
> color: black;
> border-color: black;
> }
> 
> .submit:hover {
> background-color: #1B99BF
> }

This is super helpful; thanks so much. I could not figure out the Submit button (clearly I'm no CSS expert either) before I saw this.

Below is a sample of my full code to style the whole box. I'm **_sure_** this is not 100% technically correct, but it achieved the result I'm looking for. You can tweak as necessary with whatever other values you'd like to add; I feel like it's pretty intuitive. 


### Where to Insert the Code
> Go to the Elementor Site Settings -> Custom CSS. Copy this CSS in there and change the colors to taste:

I pasted this into the Custom CSS (under advanced widget settings) just on the Post Comments widget on my Single Post Template, rather than in the Elementor Site Settings. That way, it will only change the CSS on that one post type. You method will obviously work as well if you want it site-wide and don't have any other CSS `.submit` classes.

```
.submit {
    background-color: #5a5a5a;
    color: #FFFFFF;
    border-color: #5a5a5a;
    font-family: Lato, sans-serif;
    font-size: 16px;
    font-weight: 500;
    line-height: 1.5;
}


.submit:hover {
    background-color: #000000;
    color: #FFFFFF
    border-color: #000000;
    font-family: Lato, sans-serif;
    font-size: 16px;
    font-weight: 500;
    line-height: 1.5;
}


.comment-reply-title {
    font-family: Lato, sans-serif;
    text-align: center;
    font-size: 35px;
    color: #000000;
    text-transform: uppercase;
    letter-spacing: 3px;
    line-height: 1.5;
    font-weight: 600;
}


.comment-form-comment {
    color: #000000;
    font-family: Lato, sans-serif;
    font-weight: 600;
    text-transform: uppercase;
    font-size: 16px;
    line-height: 2;
    letter-spacing: 1px;
}


.logged-in-as {
    color: #5a5a5a;
    font-family: Lato, sans-serif;
    font-weight: 400;
    font-size: 10px;
    text-align: center;
    line-height: 1;
    letter-spacing: 0.5px;
}


@media (max-width: 1025px) {
    .comment-form-comment {
    }
    
    .comment-reply-title {
        line-height: 1.25;
        font-size: 35px;
    }
    
    .logged-in-as {
        line-height: 2;
    }
}


@media (max-width: 767.9px) {
    .comment-form-comment {
        text-align: center;
        line-height: 3;
    }
    
    .comment-reply-title {
        line-height: 1.25;
        font-size: 35px;
    }
    
    .logged-in-as {
        line-height: 1.75;
    }
}
```

### Mobile Breakpoint Notes

The bottom 2 sections of code update only the mobile and tablet screen sizes. 

- You can do your own breakpoint `px` values based on your website.
- My mobile breaks at `768px` (Elementor default), but I found I needed to do `767.9px` instead to get the `1025px` tablet value to simultaneously work.
- The order of the `@media` tags matter, it also seems, if you want both tablet and mobile breakpoints.


Like I said, I'm not really CSS person (or code at all for that matter), so if there's a glaring error here, feel free to let me know. I just played around until I got something that gave the result I wanted.

**Comment by Ayadorim** [comment]
Thanks Isaac,
After I wrote the little bit of CSS to edit the Submit button, I found out that Buttons and Form fields can be styled through Elementor's Site Settings (Buttons | Form fields). That would at least help narrow down the need for extra CSS.
Greetz,
Aya


-----Original Message-----

**Comment by vojtech-richter** [comment]
Style it through css is the least important thing. Much more important is for instance fact that you cannot move with the fields, cant change content etc. Just wtf

**Comment by koolkarma** [comment]
Agree. Wordpress is a blog software and why in the world is it not possible to style the comments section??? Please add this to the list!

**Comment by aaliyahfraser** [comment]
ElementsReady is an addon for Elementor With 850+ preset elements there are over 100 free and premium widgets available
Build WooCommerce shops with ShopReady 105+ free & premium widgets with 22+ modules
This December 2022 Quomodosoft is offering 80% off
**[Grab the deals now](https://quomodosoft.com/deals/)**
Or, **[Want to join an affiliate](https://quomodosoft.com/affiliate-policy/)**

**Comment by saawood13** [comment]
I am trying to figure out how to change the color of the divider between comments with custom css. Can anyone provide me with the how to do that?

**Comment by thelockspecialistltd** [comment]
However I've figured out the way to hide any of the field by doing some CSS coding. I wanted to hide "Website" field which is successfully done. Here is the CSS code:

label {
    cursor: default;
	display:none !important;
}

In the same way we can customise this "Leave a comment" form.
